content/blog/2018-year-at-a-glance/index.md

```diff
@@ -26,9 +26,9 @@ sizes spread across the globe. The community is now a warm, welcoming, and colla
 open source contributions and passionate community leaders emerging who are helping one another while also
 helping us to make Pulumi even better.
 
-**Support for major public, private, and hybrid clouds.** We launched with support for the three major public clouds,
-[AWS](https://pulumi.io/quickstart/aws/), [Azure](https://pulumi.io/quickstart/azure/), and [Google Cloud](https://pulumi.io/quickstart/gcp/),
-and have since added capabilities across all
+**Support for major public, private, and hybrid clouds.** We launched with support for the three major public
+clouds, [AWS](https://pulumi.io/quickstart/aws/), [Azure](https://pulumi.io/quickstart/azure/), and
+[Google Cloud](https://pulumi.io/quickstart/gcp/), and have since added capabilities across all
 of them, including adding [serverless](https://pulumi.io/quickstart/aws/tutorial-rest-api.html),
```

Comment:
Just a note, this is now at `/docs/reference/tutorials/aws/tutorial-rest-api.md`

Reply:
Thanks. I did the `pulumi.io` -> `{{< ref >}}` change after wave X of posts, so I guess I missed a few. I'll do another pass to correct these.